src/native-common/Image.tsx

```diff
@@ -179,10 +195,27 @@ export class Image extends React.Component<Types.ImageProps, Types.Stateless> im
         if (this.props.headers) {
             source.headers = this.props.headers;
         }
+        if (this._forceCache) {
+            source.cache = 'force-cache';
+        }
 
         return source;
     }
 
+    private _shouldForceCacheOnError(): boolean {
+        if (Platform.getType() !== 'ios') {
+            return false;
+        }
+        if (this.props.headers) {
+            for (let key in this.props.headers) {
+                if (key.toLowerCase() === 'cache-control' && this.props.headers[key].toLowerCase() === 'max-stale') {
```

Comment:
The spec indicates I can also provide a value
`max-stale=<x seconds>`
In this case, it seems you would ignore it.  At the very least you should document the divergence in behaviour from the spec 
https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Cache-Control

Reply:
Yes, since I don't have access to cache internals I can't easily implement the time bounded variant. I'll add a comment / update docs.